src/components/homepage/HomePage.js

```diff
@@ -90,7 +90,7 @@ export function HomeContent() {
       />
       <Container className={classes.container}>
         <img src={Logo} width={"50%"}></img>
-        <Title className={classes.title}>a concept to incorporate operational experience into software projects</Title>
+        <Title className={classes.title}>Building an all-Open Source community cloud to learn things and make things better</Title>
```

Comment:
maybe, later, make `things` a link to a page that explains what are the `things` we're talking about?
and to remove duplication maybe `learn stuff and make things better`?

Reply:
maybe link to the about page, like 

`<Link to="/about">things</Link>`

Reply:
```suggestion
        <Title className={classes.title}>Building an all-Open Source community cloud to learn stuff and make <Link to="/about">things</a> better</Title>
```